app/views/mailers/notify_mailer/new_badge_email.html.erb

```diff
@@ -25,12 +25,15 @@
       Check out your profile
     </a>
   </p>
-  <hr style="opacity:0.4;width: 150px;margin: 30px auto;" />
-  <p>
-    You also get <strong><a href="<%= credits_url %>">5 new credits</a></strong> to use for <strong><a href="<%= listings_url %>">community listings</a></strong><br />
-    if you have anything you'd like to promote. 🎉
-  </p>
-  <p>
-    <strong><a href="<%= app_url(about_listings_url) %>">More information about listings<a></strong>
-  </p>
+
+  <% unless @badge.credits_awarded.zero? %>
```

Comment:
**suggestion (non-blocking):** Positive conditionals are often easier to read, so maybe this could be an alternative:

```suggestion
  <% if @badge.credits_awarded.positive? %>
```